cartography/data/jobs/analysis/gcp_compute_asset_inet_exposure.json

```diff
@@ -1,41 +1,41 @@
 {
   "statements": [
```

Comment:
are all the changes in this file just formatting changes? In that case, we should just revert back. It would make reviewing the file difficult. because of the formatting changes.